ovs/extensions/hypervisor/apis/vmware/sdk.py

```diff
@@ -903,13 +903,82 @@ def get_nfs_datastore_object(self, ip, mountpoint, filename, host=None):
                     return vm, mapping[datastore.name][filename]
         raise RuntimeError('Could not locate given file on the given datastore')
 
+    @authenticated()
+    def create_disk(self, ip, mountpoint, disk_path, size, host=None, wait=True):
+        esxhost = self._validate_host(host)
+        datastore = self.get_datastore(ip, mountpoint, host=esxhost)
+        if not datastore:
+            raise RuntimeError('Could not find datastore')
+        if self.file_exists(ip, mountpoint, disk_path):
+            raise RuntimeError('Disk {0} already exists'.format(disk_path))
+
+        disk_url = '[{0}] {1}'.format(datastore.name, disk_path)
+        config = self._client.factory.create('ns0:FileBackedVirtualDiskSpec')
+        config.adapterType = "busLogic"
+        config.diskType =  "preallocated"
+        config.capacityKb =  size * 1024 * 1024
```

Comment:
It's easier to pass size in GB. Also the kvm extension uses GB. Also the API uses size in GB.